glossary.tex

```diff
@@ -14,12 +14,14 @@
 
 \newglossaryentry{cash basis}{
   name={cash basis},
-  description={Income is deemed earned when the customer pays it, and the expenses 
+  description={\index{cash basis}
+  	Income is deemed earned when the customer pays it, and the expenses 
         are deemed incurred when the business pays them}}
 
 \newglossaryentry{contra}{
         name={contra},
-        description={A contra account is used in a general ledger to reduce the value of a related account when the two are netted together. A contra account's natural balance is the opposite of the associated account. If a debit is the natural balance recorded in the related account, the contra account records a credit.}}
+        description={\index{contra}
+        	A contra account is used in a general ledger to reduce the value of a related account when the two are netted together. A contra account's natural balance is the opposite of the associated account. If a debit is the natural balance recorded in the related account, the contra account records a credit.}}

```

Comment:
Glossary entries have a period added automatically. So the last period needs to be removed.

Reply:
Done. And the rest of them too.